webapp/components/signup/components/signup_email.jsx

```diff
@@ -0,0 +1,509 @@
+// Copyright (c) 2015 Mattermost, Inc. All Rights Reserved.
```

Comment:
New files should have the copyright year as 2016, can you update this in all new files?